db/migrations/state/0005.sql

```diff
@@ -0,0 +1,22 @@
+-- +migrate Down
+DROP TABLE IF EXISTS state.monitored_txs;
```

Comment:
Maybe eth tx manager should have its own schema / database

Reply:
we can't because we depend on reorgs and the database tx needs to be handled together to avoid out-of-sync data.